collect_app/src/test/java/org/odk/collect/android/storage/migration/StorageMigratorTest.java

```diff
@@ -27,45 +31,44 @@
     private final StorageMigrationRepository storageMigrationRepository = mock(StorageMigrationRepository.class);
     private final GeneralSharedPreferences generalSharedPreferences = mock(GeneralSharedPreferences.class);
     private final ReferenceManager referenceManager = mock(ReferenceManager.class);
+    private final BackgroundWorkManager backgroundWorkManager = mock(BackgroundWorkManager.class);
 
     @Before
     public void setup() {
-        storageMigrator = spy(new StorageMigrator(storagePathProvider, storageStateProvider, storageEraser, storageMigrationRepository, generalSharedPreferences, referenceManager));
+        whenFormDownloaderIsNotRunning();
+        whenFormUploaderIsNotRunning();
 
-        doNothing().when(storageMigrator).reopenDatabases();
         doNothing().when(storageEraser).clearOdkDirOnScopedStorage();
```

Comment:
These `doNothing` calls shouldn't be needed because `storageEraser` is a mock. It's needed below for the `reopenDatabases` because `storageMigrator` is a `spy` and not all of its methods can be run. CC @grzesiek2010

Reply:
Right probably I used spies then refactored to mocks and forgot to update here.